aptos-move/move-examples/on_chain_dice/sources/dice.move

```diff
@@ -0,0 +1,21 @@
+module module_owner::dice {
+    use std::signer::address_of;
+    use std::vector;
+    use aptos_framework::randomness;
+
+    struct DiceRollHistory has key {
+        rolls: vector<u64>,
+    }
+
+    entry fun roll(account: signer) acquires DiceRollHistory {
```

Comment:
needs private entry?

Reply:
i thought this is private entry already?